oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -101,9 +106,43 @@
     public LinuxOSProcess(int pid) {
         super(pid);
         this.commandLine = FileUtil.getStringFromFile(String.format(ProcPath.PID_CMDLINE, pid));
+        this.arguments =
+                parseArguments(FileUtil.readAllBytes(String.format(ProcPath.PID_CMDLINE, pid)));
+        this.environmentVariables =
+                parseEnvironmentVariables(FileUtil.readAllBytes(String.format(ProcPath.PID_ENVIRON, pid)));
         updateAttributes();
     }
 
+    private static List<String> parseArguments(byte[] cmdline) {
```

Comment:
This would be better moved to `ParseUtil` with a more generic name.  (Maybe `parseByteArrayToStringList`?)